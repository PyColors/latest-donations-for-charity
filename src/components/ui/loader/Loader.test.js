import React from "react";
import Loader from "./Loader";

test("Render a div", () => {
  const wrapper = shallow(<div>Hello Jest!</div>);
  expect(wrapper).toMatchSnapshot();
});

