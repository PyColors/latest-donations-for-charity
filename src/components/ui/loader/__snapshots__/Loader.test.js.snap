// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render a div 1`] = `
<div>
  Hello Jest!
</div>
`;
